model/registry/pipeline.yaml

```diff
@@ -0,0 +1,55 @@
+groups:
+  - id: registry.pipeline
+    prefix: pipeline
+    type: attribute_group
+    brief: >
+     This group describes attributes specific to pipelines.
+    attributes:
+      - id: name
```

Comment:
do we need an attribute to identify pipeline provider (`Azure DevOps`, `GitHub Actions`, `CircleCI`), etc?

This would affect naming or id format and could provide additional context to the telemetry consumer (e.g. they can show an icon or build solution-specific experience)

suggesting `*.system` to match similar attributes in db, messaging, gen_ai

Reply:
#1193 - will define as part of that issue